### PR TITLE
Sheet: Add overflow for long content and remove footer

### PR DESCRIPTION
**Pull request by jialin-he**
**This change:** (check at least one)

- [ ] Adds a new feature
- [x] Fixes a bug
- [ ] Improves maintainability
- [ ] Improves documentation
- [ ] Is a release activity

**Is this a breaking change?** (check one)

- [ ] Yes
- [x] No

**Is the:** (complete all)

- [x] Title of this pull request clear, concise, and indicative of the issue number it addresses, if any?
- [ ] Test suite(s) passing?
- [ ] Code coverage maximal?
- [ ] Changeset added?
- [ ] Component status page up to date?

**What does this change address?**
Fix the sheet to allow it scroll with long content.

Before:
image

After:
image

**How does this change work?**
Add overflow and other css constraints.
Also remove the header and footer from sheet to match what we did for popover.

### CODE REVIEW

File: packages/pharos/src/components/sheet/pharos-sheet.scss [hunk 1]
```diff
@@ -77,6 +77,9 @@
 
   .sheet__wrapper {
     padding: 0 var(--pharos-spacing-one-and-a-half-x);
+    display: flex;
+    flex-direction: column;
+    height: calc(100% - 67px);
```
Comment (daneah) [inline packages/pharos/src/components/sheet/pharos-sheet.scss:82]: Where does this `67px` value arise from?

Comment (chrisjbrown) [reply packages/pharos/src/components/sheet/pharos-sheet.scss:82]: is specifying the height necessary? doesn't seem to effect it if i remove it
Screenshot 2023-09-14 at 4 05 08 PM

Comment (chrisjbrown) [reply packages/pharos/src/components/sheet/pharos-sheet.scss:82]: ah nvm. i see it's needed for overflow scrolling 🤔

Comment (chrisjbrown) [reply packages/pharos/src/components/sheet/pharos-sheet.scss:82]: does replacing the `height` with `overflow-y: scroll` achieve the same result?
Screenshot 2023-09-14 at 4 20 37 PM

Comment (jialin-he) [reply packages/pharos/src/components/sheet/pharos-sheet.scss]: Thank you for the advice! After discussing with Lori we now decide to actually remove the footer from the sheet (at least for now) to match the treatment we have in the popover. So this problem naturally solves.

**Review by chrisjbrown — APPROVED** [review]

**Pull request merged**